mh/pat.py

```diff
@@ -1469,12 +1479,22 @@ def sendNtcLayerChat(self, unk1, info, message, seq):
         The server sends a chat message.
         """
         data = struct.pack(">B", unk1)
-        info.text_color = pati.Long(0xbb3385ff)
-        info.sender_id = pati.String(b"C1I2D3")
-        info.sender_name = pati.String(b"Cid")
+
+        info.text_color = pati.Long(LAYER_CHAT_COLORS[self.session.layer])
+        info.sender_id = pati.String(self.session.capcom_id.encode("utf-8"))
```

Comment:
There should be a `TODO` regarding the japanese version using `Shift-JS`

Reply:
Actually I don't think that this is a needed? We are not doing it in any other place that we send the `capcom_id`.

Reply:
I think it's unneeded as I used a helper in `pati.String` and `pati.Binary` to avoid encoding shenanigans. You can leave it as is, I'll address it in a follow-up PR.